diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0247a0aca0..2c5f6ddc68 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -330,6 +330,13 @@ endif()

 add_sdks()
 
+SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "AWS C++ SDK")
+SET(CPACK_PACKAGE_VENDOR "Amazon")
+SET(CPACK_PACKAGE_VERSION "${PROJECT_VERSION}")
+SET(CPACK_GENERATOR "RPM;DEB")
+SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "Package fairy")
+INCLUDE(CPack)
+
 # for user friendly cmake usage
 include(setup_cmake_find_module)
 